build_compilers/lib/src/modules.dart

```diff
@@ -5,26 +5,55 @@
 import 'package:analyzer/dart/element/element.dart';
 import 'package:build/build.dart';
 
+/// A collection of Dart libraries in a strongly connected component and the
```

Comment:
nit: `Dart files`, it may include part files etc right?

Also we should probably add some testing around part files?

Reply:
Filed https://github.com/dart-lang/build/issues/393

I think we should cover part files in a followup.